cluster-autoscaler/Dockerfile

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
In general, I suggest to follow the best practice from the other projects in the `gardener` org and build the binaries directly in the container and use a multi-stage build in this file.